fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/agg/HllUnion.java

```diff
@@ -32,7 +32,11 @@
 
 /** HllUnion */
 public class HllUnion extends AggregateFunction
-        implements UnaryExpression, PropagateNullable, ImplicitCastInputTypes {
+        implements UnaryExpression, AlwaysNotNullable, ExplicitlyCastableSignature {
+    public static final List<FunctionSignature> SIGNATURES = ImmutableList.of(
+            FunctionSignature.ret(HllType.INSTANCE).args(HllType.INSTANCE)
+    );
+
     public HllUnion(Expression arg0) {
         // TODO: change to hll_union in the future
```

Comment:
add a reason to comment?

Reply:
In the past, we can not find hll_union in frontend's FunctionSet.
Now, we don't need to find function in FunctionSet, so hll_union is available